cmd/auth/auth.go

```diff
@@ -68,11 +69,20 @@ var AuthCmd = &cobra.Command{
 			password = strings.TrimSpace(string(bytePassword))
 		}
 
+		baseURL := ""
+		if backend == "llama" {
+			fmt.Printf("Enter %s API Base URL (e.g. `http://localhost:8080/v1`): ", backend)
+			scanner := bufio.NewScanner(os.Stdin)
```

Comment:
shoudn't we use the following to be more consistent ?

```go
bytePassword, err := term.ReadPassword(int(syscall.Stdin))
			if err != nil {
				color.Red("Error reading %s Key from stdin: %s", backend,
					err.Error())
				os.Exit(1)
			}
			password = strings.TrimSpace(string(bytePassword))
```

Reply:
I was thinking to that indeed, but then the baseurl is not something should be kept confidential and be hidden to the user, so I went with a simpler approach. Happy to revisit this

Reply:
My take is having the baseURL as another flag, a) it's not sensitive data and b) the rest of non sensitive information get passed via flags, I have implemented that in my draft PR to showcase it

Reply:
@arbreezy looks great! I really like the approach :+1:

Reply:
make sense 👍